docs/tutorials.superdense_coding.rst

```diff
@@ -1,4 +1,159 @@
 Superdense Coding
 ==================
 
-Coming soon.
\ No newline at end of file
+In classical communication, sending two bits of information requires transmitting
+two physical bits. But with the help of quantum mechanics we can bend this rule
+using the phenomenon of quantum entanglement. **Superdense coding** proposed by
```

Comment:
```suggestion
two physical bits. But with the help of quantum entanglement, we can bend this rule. **Superdense coding** proposed by
```

Reply:
move the line `**Superdense coding** proposed by` and everything after this to a new paragraph.